docs/java.md

```diff
@@ -0,0 +1,373 @@
+                            row.getString("album"),
+                            row.getTimestamp("created_at").toString()
+                    )
+            );
+        }
+     }
+}
+```
+
+
+### 3.4 Updating Data
+
+Ok, almost there! Now we're going to learn about `UPDATE` but here's a disclaimer: 
+> INSERT and UPDATES are not the same!
+
+There's a myth in Scylla/Cassandra community that it's the same for the fact that you just need the `Partition Key` and `Clustering Key` (if you have one) and query it.
+
+Read more about [`INSERT` and `UPDATE`](https://docs.scylladb.com/stable/using-scylla/cdc/cdc-basic-operations.html)
+
+As you can see, the `UPDATE` query takes two fields in the `WHERE` clause (PK and CK). Check the snippet below: 
+
+
+```java
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.BoundStatement;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.PlainTextAuthProvider;
+import com.datastax.driver.core.policies.DCAwareRoundRobinPolicy;
+
+import java.util.Date;
+import java.util.Scanner;
+
+class Main {  
+  
+    public static void main(String[] args) { 
+        Cluster cluster = Cluster.builder()  
+            .addContactPoints("your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud", "your-node-url.7207ca5a8fdc45f2b03f.clusters.scylla.cloud")  
+            .withLoadBalancingPolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("AWS_US_EAST_1").build()) // your local data center  
+            .withAuthProvider(new PlainTextAuthProvider("scylla", "*******"))  
+            .build();  
+            
+            Session session = cluster.connect();  
+
+            PreparedStatement statement = session.prepare(
+                "UPDATE songs set title = ? where id = ? AND updated_at = ?"
+            );
+
+            BoundStatement bound = statement.bind()
+                    .setLong(0, 1)
+                    .setString(1, "Rock and Roll")                    
+                    .setTimestamp(2, new Date());
+
+            session.execute(bound);
+     }
+}
+```
+
+After the data gets inserted, query all columns and filter by the ID:
+```
+scylla@cqlsh:media_player> select * from songs where id = 1;
+
+id | updated_at                      | album       | artist | created_at                      | title
+---+---------------------------------+-------------+--------+---------------------------------+----------------------------
+ 1 | 2023-03-02 22:00:00.000000+0000 | Smithereens |   Joji | 2023-03-02 22:00:00.000000+0000 |              Glimpse of Us
+ 1 | 2023-03-02 23:10:00.000000+0000 |        null |   null |                            null | Glimpse of US - Inutilismo
+```
+
+It only updated the field `title` and `updated_at` (the Clustering Key) and since we didn't input the rest of the data, it will not be replicated as expected.
```

Comment:
```suggestion
It only updated the field `title` and `updated_at` (the Clustering Key), and since we didn't input the rest of the data, it will not be replicated as expected.
```